include/oneapi/dpl/pstl/utils.h

```diff
@@ -584,10 +584,10 @@ struct __lifetime_keeper : public __lifetime_keeper_base
 // Bitwise type casting, same as C++20 std::bit_cast
 template <typename _Dst, typename _Src>
 ::std::enable_if_t<
-    sizeof(_Dst) == sizeof(_Src) && ::std::is_trivially_copyable_v<_Dst> && ::std::is_trivially_copyable_v<_Src>, _Dst>
+    sizeof(_Dst) == sizeof(_Src) && ::std::is_trivially_copyable_v<_Dst>&& ::std::is_trivially_copyable_v<_Src>, _Dst>
 __dpl_bit_cast(const _Src& __src) noexcept
 {
-#if __cpp_lib_bit_cast >= 201806L || _ONEDPL___cplusplus >= 202002L && __has_include(<bit>)
+#if __cpp_lib_bit_cast >= 201806L && __has_include(<bit>)
```

Comment:
In combination with the feature macro (instead of the C++ version check, `__has_include(<bit>)` makes no sense I think, because the feature macro implies that the header should be available.